### DISCUSSION TITLE
Permission Issue w/ 3.12.4

### DISCUSSION THREAD

**Discussion by zoenglinghou** [docker-library/rabbitmq#667] [Q&A] [answered]
We had a permission issue running `rabbitmq` 3.12.4, updating from some version of `3.11` (no persistent storage):

```
Failed to create thread: Operation not permitted (1) Aborted (core dumped)
```

It happens across different docker versions, and the container is started by root, unprivileged. Is it a known issue?

Thanks a lot for your help

**Comment by lukebakken** [comment, marked as answer]
GitHub provides a feature where you can search a repository -

https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests

In your case, I searched the string `Failed to create thread` and found this relevant issue:

https://github.com/docker-library/rabbitmq/issues/650

Please read it to see if that fixes the issue in your environment.

**Comment by zoenglinghou** [comment]
Thank you for pointing out the solution. I would dig deeper into the issue you link. I focused my search and debugging too much with the permission issue.
